09_proxmox_fstrim_cloudinit_ansible_lvm_08-sep-2022/cloudinit-exp/cloudinit.md

```diff
@@ -53,8 +54,9 @@ qm set 6013 --ipconfig0 ip=10.44.106.13/24,gw=10.44.111.1
 
 You have to edit parameters accordingly to your preferences and settings on your machine.
```

Comment:
Please also avoid using you:

It is necessary to edit parameters accordingly to the desired preferences and settings on the machine.